src/SprykerEco/Yves/Payone/Plugin/Router/PayoneRouteProviderPlugin.php

```diff
@@ -178,6 +181,20 @@ protected function addPayoneExpressCheckoutLoadDetailsRoute(RouteCollection $rou
         return $routeCollection;
     }
 
+    /**
+     * @param \Spryker\Yves\Router\Route\RouteCollection $routeCollection
```

Comment:
* Add a `@uses` tag that points to a controller action.

Reply:
++